roles/kubernetes-apps/metrics_server/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+metrics_server_namespace: "kube-system"
+metrics_server_kubelet_insecure_tls: true
+metrics_server_kubelet_preferred_address_types: "InternalIP"
+metrics_server_metric_resolution: 60s
+base_metrics_server_cpu: 40m
+base_metrics_server_memory: 35Mi
```

Comment:
Please rename this variable to `metrics_server_memory_limit`

Reply:
done